File: java/program/Crocodile.java
public class Crocodile extends Reptile {
    private int biteForce;

    public Crocodile(String name, double length, String habitat, String diet, int biteForce) {
        super(name, length, habitat, diet);
        this.biteForce = biteForce;
    }

    @Override
    public void display() {
        System.out.println("Crocodile: " + name +
                           ", Bite Force: " + biteForce + " N, Habitat: " + habitat);
    }
}

File: cpp/Crocodile.cpp
#include "Crocodile.h"

Crocodile::Crocodile(string n, double l, string h, string d, int bf)
    : Reptile(n, l, h, d), biteForce(bf) {}

void Crocodile::display() {
    cout << "Crocodile: " << name
         << ", Bite Force: " << biteForce
         << " N, Habitat: " << habitat << endl;
}
